### DISCUSSION TITLE
Post Tweets on my timeline from specific accounts

### DISCUSSION THREAD

**Discussion by TownyMontana** [tweepy/tweepy#1836] [Questions] [answered]
How do I post tweets from specific users on my timelime. Everytime a specific user posts, I want tweepy to post that on my timeline after 5 - 10 minutes.

**Comment by Harmon758** [comment, marked as answer]
You can retrieve them using [streaming](https://docs.tweepy.org/en/v4.6.0/streaming.html) or [`API.user_timeline`](https://tweepy.readthedocs.io/en/v4.6.0/api.html#tweepy.API.user_timeline) / [`Client.get_users_tweets`](https://tweepy.readthedocs.io/en/v4.6.0/client.html#tweepy.Client.get_users_tweets), and you can Tweet using [`API.update_status`](https://tweepy.readthedocs.io/en/v4.6.0/api.html#tweepy.API.update_status) / [`Client.create_tweet`](https://tweepy.readthedocs.io/en/v4.6.0/client.html#tweepy.Client.create_tweet).
